sdk/extensions/chronopost/karrio/mappers/chronopost/proxy.py

```diff
@@ -0,0 +1,55 @@
+import typing
+import urllib.parse
+import karrio.lib as lib
+import karrio.api.proxy as proxy
+from karrio.core.utils import (
+    Envelope,
+    XP,
+    request as http,
+    exec_parrallel,
+)
```

Comment:
```suggestion
```

I would suggest using `lib` instead because it has all these helpers and it is better typed.